modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -3,21 +3,29 @@ ifndef::env-site,env-github[]
+WARNING: A Scatter-Gather component cannot be configured with only one route. A Mule application with this configuration throws an exception and does not start, at least two routes are necessary in a Scatter-Gather configuration to be valid.
 
-To illustrate how this works, let’s look at two cases. In the first case, the routes in a Scatter-Gather component each contain a Try scope. One of the routes generates an error that is successfully handled by that route’s Try scope, so the route is still able to generate a Mule event. The Scatter-Gather component consolidates the Mule events from all routes into a new Mule event, as usual, and passes the consolidated event to the next event processor.
+== Error Handling Inside Scatter-Gather Routes
 
-In the second case, an error occurs in a route. This route either does not contain a Try scope or the error is of a type that the Try scope cannot handle. Although the other routes all complete, the one failed route causes the Scatter-Gather component to throw a `MULE.COMPOSITE ERROR` error, which the flow passes to the Error Handler component that you have configured for your flow. The error-handling event processors are able to process the Mule events from the completed routes.
+You can use a Try scope in each route of a Scatter-Gather component to handle any errors that might be generated by a route’s event processors. Unhandled errors that are thrown because of a problem in a route cause the Scatter-Gather component to also throw an error of type `MULE:COMPOSITE_ERROR`. The component throws this error only after every route completes, either with a successful execution or with an error that causes the route to fail. Event processing does not proceed further than the Scatter-Gather component in the flow. Instead, the flow branches to your error-handling event processors. The `MULE:COMPOSITE_ERROR` error object gathers together not only the errors from the route that failed, but also the Mule events from completed routes, so that your application can still process Mule events from the routes that completed by using your error-handling event processors.
+
+To illustrate how this works, consider the following two cases:
+
+* The routes in a Scatter-Gather component each contain a Try scope. One of the routes generates an error that is successfully handled by that route’s Try scope, so the route is still able to generate a Mule event. The Scatter-Gather component consolidates the Mule events from all routes into a new Mule event, as usual, and passes the consolidated event to the next event processor.
```

Comment:
only if the error handler within the route is `on-error-continue`. For `on-error-propagate` an error will be generated for the route still.